apps/web/pages/[slug]/index.ts

```diff
@@ -1,9 +1,10 @@
-import '@island.is/api/mocks'
 import { getServerSidePropsWrapper } from '@island.is/web/utils/getServerSidePropsWrapper'
 
 import withApollo from '../../graphql/withApollo'
 import { withLocale } from '../../i18n'
 import articleScreen from '../../screens/Article'
+
+import '@island.is/api/mocks'
```

Comment:
I'm not sure if moving the api mocks import has any effect as I've never used mocks myself.